tests/platform_tests/daemon/test_ledd.py

```diff
@@ -122,11 +122,7 @@ def test_pmon_ledd_term_and_start_status(check_daemon_status, duthosts, rand_one
 
     duthost.stop_pmon_daemon(daemon_name, SIG_TERM, pre_daemon_pid)
 
-    daemon_status, daemon_pid = duthost.get_pmon_daemon_status(daemon_name)
-    pytest_assert(daemon_status != expected_running_status and pre_daemon_pid != daemon_pid,
-                          "{} status for SIG_TERM should not be {} with pid:{}!".format(daemon_name, daemon_status, daemon_pid))
-
-    time.sleep(10)
+    time.sleep(15)
```

Comment:
why not use wait_until?

Reply:
@lguohan  I updated the pr. please review again.